fluentd-daemonset-elasticsearch-rbac.yaml

```diff
@@ -89,6 +89,12 @@ spec:
             value: "elastic"
           - name: FLUENT_ELASTICSEARCH_PASSWORD
             value: "changeme"
+          # Prevents circular logging issue
+          - name: FLUENT_CONTAINER_TAIL_EXCLUDE_PATH
+            value: /var/log/containers/fluent*
+          # Docker log parser
+          - name: FLUENT_CONTAINER_TAIL_PARSER_TYPE
+            value: /^(?<time>.+) (?<stream>stdout|stderr)( (?<logtag>.))? (?<log>.*)$/
```

Comment:
It's not correct, it not valid for `@type` here:
https://github.com/fluent/fluentd-kubernetes-daemonset/blob/4066426d1a23da07e7f014ae0ca9693fdd1655a8/templates/conf/tail_container_parse.conf.erb#L2

It shoud be `cri` as described in https://github.com/fluent/fluentd-kubernetes-daemonset/issues/434#issuecomment-831801690